### PR TITLE
Adding the CNAME Setup limitation

### PR DESCRIPTION
**Pull request by lougayou**
The Dev Docs are not mentioning the limitations associated to the CNAME Setup, like we used to have in the KB SPM-1486

### CODE REVIEW

File: content/dns/zone-setups/partial-setup/_index.md [hunk 1]
```diff
@@ -28,3 +28,10 @@ With a partial zone, Cloudflare resolves [DNS records differently](/dns/zone-set
 A partial (`CNAME`) setup requires the proxied hostname to be pointed to Cloudflare via a `CNAME` record. Since [`CNAME` records are not allowed on the zone apex](https://datatracker.ietf.org/doc/html/rfc1912#section-2.4) (`example.com`), you can only proxy your zone apex to Cloudflare if your authoritative DNS provider supports [`CNAME` Flattening](/dns/additional-options/cname-flattening/).
 
 If your authoritative DNS provider does not support CNAME Flattening, redirect its traffic — for example, with an `.htaccess` file — to a subdomain proxied to Cloudflare.
+  
+### Limitations
+
+The CNAME setup has two limitations:
```
Comment (hannesb0) [inline content/dns/zone-setups/partial-setup/_index.md]: Let's stick with "partial"

File: content/dns/zone-setups/partial-setup/_index.md [hunk 2]
```diff
@@ -28,3 +28,10 @@ With a partial zone, Cloudflare resolves [DNS records differently](/dns/zone-set
...
+
+* DDOS protection for attacks against DNS infrastructure is only available for the delegated subdomain records.
```
Comment (hannesb0) [inline content/dns/zone-setups/partial-setup/_index.md]: ~DDOS~ DDoS protection for attacks against DNS infrastructure is only available ~for the delegated subdomain records.~ for domains on full setup. Domains on the partial setup are not using Cloudflare authoritative nameservers.

File: content/dns/zone-setups/partial-setup/_index.md [hunk 3]
```diff
@@ -28,3 +28,10 @@ With a partial zone, Cloudflare resolves [DNS records differently](/dns/zone-set
...
+* Only subdomains, not the root domain, can use Cloudflare's services. This limitation is imposed by Internet DNS specifications.
```
Comment (hannesb0) [inline content/dns/zone-setups/partial-setup/_index.md]: I would replace this bullet point by something like this:
* In order to proxy traffic through the root of your domain (like `example.com`), your authoritative DNS provider needs to support CNAME flattening, so you can point your zone apex to Cloudflare. This restriction is imposed by the general limitation that you cannot add a CNAME record on your zone apex (see [RFC 1034 section 3.6.2](https://datatracker.ietf.org/doc/html/rfc1034#section-3.6.2) and [RFC 1912 section 2.4](https://www.rfc-editor.org/rfc/rfc1912#section-2.4)). Alternatively, [static IPs or BYOIPs](https://developers.cloudflare.com/fundamentals/get-started/concepts/cloudflare-ip-addresses/#customize-cloudflare-ip-addresses) can be used.

Comment (kodster28) [reply content/dns/zone-setups/partial-setup/_index.md]: @hannesb0, this section was actually partially duplicating the `CNAME flattening` section above. Tried combining them but may have missed or misstated something.

**Review by kodster28 — APPROVED** [review]

**Pull request merged**